andela/ah-nyati-backend
# #166983634 Fix build script

#### What does this PR do?
- Fixes `build` script

#### Description of Task to be completed?
- Restructures application folder structure
- Rewrite build script using the correct directory

#### How should this be manually tested?
- In the terminal, run the following command: `npm start`
- The server should run successfully

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- #166983634
- https://www.pivotaltracker.com/story/show/166983634

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
